data/i18n/bn/bn.toml

```diff
@@ -252,22 +341,30 @@ other = "অথবা"
 other = "একটি সমস্যা রিপোর্ট করুন"
 
 [layouts_docs_partials_feedback_thanks]
-other = "সাহায্য করার জন্য ধন্যবাদ. কুবারনেটস কীভাবে ব্যবহার করবেন সে সম্পর্কে আপনার যদি একটি নির্দিষ্ট, উত্তরযোগ্য প্রশ্ন থাকে তবে এটি জিজ্ঞাসা করুন"
+other = "সাহায্য করার জন্য ধন্যবাদ. কুবারনেটিস কীভাবে ব্যবহার করবেন সে সম্পর্কে আপনার যদি একটি নির্দিষ্ট, উত্তরযোগ্য প্রশ্ন থাকে তবে এটি জিজ্ঞাসা করুন"
 
 [layouts_docs_search_fetching]
 other = "ফলাফল আনা হচ্ছে..."
 
+[legacy_repos_message]
+other = """The legacy package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`) have been
+[deprecated and frozen starting from September 13, 2023](/blog/2023/08/31/legacy-package-repository-deprecation/).
+**Using the [new package repositories hosted at `pkgs.k8s.io`](/blog/2023/08/15/pkgs-k8s-io-introduction/)
+is strongly recommended and required in order to install Kubernetes versions released after September 13, 2023.**
+The deprecated legacy repositories, and their contents, might be removed at any time in the future and without
+a further notice period. The new package repositories provide downloads for Kubernetes versions starting with v1.24.0."""
```

Comment:
```
other = """লিগেসি প্যাকেজ রিপোজিটরিজগুলি (`apt.kubernetes.io` এবং `yum.kubernetes.io`)
[১৩ সেপ্টেম্বর, 2023 থেকে বন্ধ করা হয়েছে এবং ফ্রযেন করা হয়েছে](/blog/2023/08/31/legacy-package-repository-deprecation/)।
**[`pkgs.k8s.io`-এ হোস্ট করা নতুন প্যাকেজ রিপোজিটরিজগুলি](/blog/2023/08/15/pkgs-k8s-io-introduction/) ব্যবহার
করা দৃঢ়ভাবে সুপারিশ করা হয় এবং ১৩ সেপ্টেম্বর, ২০২৩-এর পরে প্রকাশিত কুবারনেটিস ভার্সনগুলি ইনস্টল করার জন্য প্রয়োজনীয়।**
অপ্রচলিত লিগেসি  রিপোজিটরিজগুলি, এবং তাদের কনটেন্ট, ভবিষ্যতে যেকোনো সময় সরানো হতে পারে এবং পরবর্তী
নোটিশের সময় ছাড়াই। নতুন প্যাকেজ রিপোজিটরিজগুলি v1.24.0 থেকে শুরু করে কুবারনেটিস ভার্সনগুলির জন্য ডাউনলোড প্রদান করে।"""
```

Reply:
if the blogs are localized then add `(/bn/blog/2023/08/31/legacy-package-repository-deprecation/)।`

Reply:
https://github.com/kubernetes/website/pull/46271/files#r1594992002 as a committable suggestion:
```suggestion
other = """লিগেসি প্যাকেজ রিপোজিটরিজগুলি (`apt.kubernetes.io` এবং `yum.kubernetes.io`)
[১৩ সেপ্টেম্বর, 2023 থেকে বন্ধ করা হয়েছে এবং ফ্রযেন করা হয়েছে](/blog/2023/08/31/legacy-package-repository-deprecation/)।
**[`pkgs.k8s.io`-এ হোস্ট করা নতুন প্যাকেজ রিপোজিটরিজগুলি](/blog/2023/08/15/pkgs-k8s-io-introduction/) ব্যবহার
করা দৃঢ়ভাবে সুপারিশ করা হয় এবং ১৩ সেপ্টেম্বর, ২০২৩-এর পরে প্রকাশিত কুবারনেটিস ভার্সনগুলি ইনস্টল করার জন্য প্রয়োজনীয়।**
অপ্রচলিত লিগেসি  রিপোজিটরিজগুলি, এবং তাদের কনটেন্ট, ভবিষ্যতে যেকোনো সময় সরানো হতে পারে এবং পরবর্তী
নোটিশের সময় ছাড়াই। নতুন প্যাকেজ রিপোজিটরিজগুলি v1.24.0 থেকে শুরু করে কুবারনেটিস ভার্সনগুলির জন্য ডাউনলোড প্রদান করে।"""
```

Reply:
can one string have 2 other ?

![image](https://github.com/kubernetes/website/assets/155321064/95a1b6d4-25e6-4658-a9a4-abe457d6b390)